src/System.Runtime.Numerics/ref/System.Runtime.Numerics.cs

```diff
@@ -153,14 +153,15 @@ namespace System.Numerics
         public static System.Numerics.BigInteger Remainder(System.Numerics.BigInteger dividend, System.Numerics.BigInteger divisor) { throw null; }
         public static System.Numerics.BigInteger Subtract(System.Numerics.BigInteger left, System.Numerics.BigInteger right) { throw null; }
         public byte[] ToByteArray() { throw null; }
+        public byte[] ToByteArray(bool isUnsigned=false, bool isBigEndian=false) { throw null; }
         public override string ToString() { throw null; }
         public string ToString(System.IFormatProvider provider) { throw null; }
         public string ToString(string format) { throw null; }
         public string ToString(string format, System.IFormatProvider provider) { throw null; }
         public static bool TryParse(string value, System.Globalization.NumberStyles style, System.IFormatProvider provider, out System.Numerics.BigInteger result) { throw null; }
         public static bool TryParse(string value, out System.Numerics.BigInteger result) { throw null; }
         public static bool TryParse(ReadOnlySpan<char> value, out System.Numerics.BigInteger result, System.Globalization.NumberStyles style = System.Globalization.NumberStyles.Integer, System.IFormatProvider provider = null) { throw null; }
-        public bool TryWriteBytes(System.Span<byte> destination, out int bytesWritten) { throw null; }
+        public bool TryWriteBytes(System.Span<byte> destination, out int bytesWritten, bool isUnsigned=false, bool isBigEndian=false) { throw null; }
```

Comment:
I thought `out` parameters were supposed to be at the end?

Reply:
`out` is "supposed to" be (design pattern).  Optional "must" be (language rule).

Reply:
Would it make more sense to add an overload here then instead of using optional parameters?  For some reason I find it super hard to read (and surprising) when a random `out` parameter is in the middle of a method call.

Reply:
Many of the span-based methods we've added now have `out` in the middle, e.g. most of the TryFormat methods we've added are of the form:
```C#
public bool TryFormat(Span<char> destination, out int charsWritten, string format = null, IFormatProvider provider = null);
```
If we're concerned about that here, we should revisit all of them together.

Reply:
If that's the established pattern, then I remove my concern.  But I still find it surprising.